ydiff.py

```diff
@@ -850,6 +860,13 @@ def _process_args(self, largs, rargs, values):
     parser.add_option(
         '', '--wrap', action='store_true',
         help='wrap long lines in side-by-side view')
+    parser.add_option(
+        '-p', '--pager', default='less', metavar='M',
```

Comment:
I think it should default to None so that it's possible to distinguish explicit --pager option from default behavior, we will keep the original behavior only when the option is not provided, this makes sure it won't break any existing user.

Reply:
But isn't `less` the default behavior?

Reply:
Explained in the other comment.